### ISSUE TITLE
Email Accessibility - Contrast too low

### ISSUE THREAD

**Issue by edent** [mysociety/fixmystreet#5348]
**Describe the bug**
Current email colours fail WCAG guidance on text contrast.

**To Reproduce**
1. Receive an email with an update.
2. Try to read the text at the top and bottom (e.g. " This email was sent automatically, from an unmonitored email account. Please do not reply to it."
4. Hurt your eyes

**Expected behavior**
The current text colour is #4E646B; the background is #42494C. This is an accessibility failure - https://www.siegemedia.com/contrast-ratio#%234E646B-on-%2342494C

**Screenshots**
![Image](https://github.com/user-attachments/assets/1d4e4640-36d0-4e3b-abac-45c4e15419ac)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Firefox
 - Version: 134.0.2 (64-bit)

**Additional context**

The problem occurs at

https://github.com/mysociety/fixmystreet/blob/ed9a79956a076f80acbac7c33ccf4b2e00b1df8e/templates/email/default/_email_settings.html#L23

Changing it to `color_blue_pale` will fix the contrast issue.

**Comment by dracos** [comment]
Thanks; perhaps we should actually drop the line entirely, it's only repeating the heading just below. I wonder if there's circumstances where it's saying something more useful, we can check.
